tidy.Rmd

```diff
@@ -62,27 +62,27 @@ Why ensure that your data is tidy? There are two main advantages:
     tools that work with it because they have an underlying uniformity.
     
 1.  There's a specific advantage to placing variables in columns because
-    it allows R's vectorised nature to shine. As you learned in
-    [mutate](#mutate-funs) and [summary functions](#summary-funs), most 
+    it allows R's vectorised nature to shine. As you learned in 
+    Sections \@ref(mutate-funs) and \@ref(summarise-funs), most 
     built-in R functions work with vectors of values. That makes transforming 
     tidy data feel particularly natural.
 
 dplyr, ggplot2, and all the other packages in the tidyverse are designed to work with tidy data. Here are a couple of small examples showing how you might work with `table1`.
 
-```{r, out.width = "50%"}
+```{r fig.width = 5, fig.alt = "This figure shows the numbers of cases in 1999 and 2000 for Afghanistan, Brazil, and China, with year on the x-axis and number of cases on the y-axis. Each point on the plot represents the number of cases in a given country in a given year. The points for each country are differentiated from others by color and shape and connected with a line, resulting in three, non-parallel, non-intersecting lines. The numbers of cases in China are highest for both 1999 and 2000, with values above 200,000 for both years. The number of cases in Brazil is approximately 40,000 in 1999 and approximately 75,000 in 2000. The numbers of cases in Afghanistan are lowest for both 1999 and 2000, with values that appear to be very close to 0 on this scale."}
```

Comment:
These are really nice illustrations of why you want both `fig.cap` and `fig.alt` 😄